pages/common/find.md

```diff
@@ -6,6 +6,10 @@
 
 `find {{root_path}} -name '{{*.ext}}'`
 
+- Find files by multiple search patterns, pattern_1 OR ending with case insensitive pattern_2:
```

Comment:
No need to introduce iname and complicate this example. We already have `-iname` below. Please just introduce `-or` by saying "Find files by matching multiple patterns:" You are also missing a * at the end of `pattern_2`.

Reply:
I omitted trailing `*` intentionally to illustrate
>OR **ending** with case insensitive pattern_2:

My intention was to show as many as possible configuration details in one example. It sounds like you would prefer more focused example instead. Works for me.